xlators/features/bit-rot/src/bitd/bit-rot.c

```diff
@@ -1712,7 +1712,7 @@ notify(xlator_t *this, int32_t event, void *data, ...)
 }
 
 static void
-br_fini_signer(xlator_t *this, br_private_t *priv)
+br_fini_signer(br_private_t *priv)
 {
     int i = 0;
 
```

Comment:
nit: the below check is not needed. - priv is not null, as it was already checked in fini()